examples/liquid.ini

```diff
@@ -217,6 +217,13 @@ wait_green_count = 6
 #                         Set to "-1" for batch processing (indexing), or "6s" for normal searching.
 #                         Defaults to "6s".
 #                         Any collection can override this value.
+# - `image_classification_enabled` - Set to "true" to run image classification service. The service can perform
+#                                    object detection and image classification
+# - `image_classification_count`    - Set the number of containers to run, default 1.
+# - `image_classification_memory_limit` - Override memory limit for container, default 900.
+# - `image_classification_waitress_threads` - Set the number of waitress threads, default 30.
+# - `image_classification_object_detection_enabled` - Set the service to enable object detection. Default is "false".
+# - `image_classification_object_detection_model`   - Set the model used for object detection. Default is "yolo".
```

Comment:
I think you're missing from here a bunch of values, like `image_classification_classify_images_enabled` and friends

Reply:
Also, for the `_model` configs, please list out all the available options, instead of just the default one. You can also put a link pointing to the readme of the NLP repo if you want to reduce duplication, but there still needs to be an extensive list of options somewhere.